con-sdl.c

```diff
@@ -132,18 +133,18 @@ static void scrollup(void)
 
 
 /* output character at cursor location*/
-void sdl_textout(unsigned char c)
+void sdl_textout(word_t c)
 {
 	cursoroff();
 
-	switch (c) {
+	switch (c & 0xFF) {
 	case '\0':	return;
 	case '\b':	if (--curx <= 0) curx = 0; goto update;
 	case '\r':	curx = 0; goto update;
 	case '\n':  goto scroll;
 	}
 
-	mem_stat[VID_BASE + (cury * COLS + curx) * 2] = c;
+	*(word_t *)&mem_stat[VID_BASE + (cury * COLS + curx) * 2] = c;
```

Comment:
Same remark as above : if using a second parameter as the character attribute, `mem_stat [+0] = char_code` and mem_stat [+1] = attr_code` would be cleaner.

Reply:
This could be changed once the entire project is evaluated for a 2nd parameter to be used for attributes.
Again, its a bit complicated because the cursor code is dependent on OR'ing in a value to code point, so would like to defer to another PR, please.